db/academyPortal_2020-05-19.sql

```diff
@@ -5,9 +5,9 @@
 # http://www.sequelpro.com/
 # https://github.com/sequelpro/sequelpro
 #
-# Host: 127.0.0.1 (MySQL 5.7.29)
+# Host: 127.0.01 (MySQL 5.7.29)
```

Comment:
This was right the first time. The IP Address for localhost is 127.0.0.1

Reply:
Not sure what's happened here to be honest. Try dumping the db again?

Reply:
fixed this, re exported db